app/routes/users/components/StudentConfirmation.tsx

```diff
@@ -1,206 +1,229 @@
-              component={RadioButton.Field}
-              inputValue="komtek"
-            />
-          </RadioButtonGroup>
-          <RadioButtonGroup
-            name="isTwoYears"
-            label={
-              <Flex alignItems="center" gap={5}>
-                <div>Begynner du på 2-årig eller 5-årig master?</div>
-                <Tooltip content="Huk av 2-årig master her kun hvis du begynner på to-arig master studiet på din respektive linje. Dvs. du har en bachelor fra før av. Begynner du i første klasse og skal studere i 5 år, huk av 5-årig master.">
-                  <Icon name="information-circle-outline" size={20} />
-                </Tooltip>
-              </Flex>
-            }
-          >
-            <Field
-              name="isTwoYearsYes"
-              label="2-årig master"
-              component={RadioButton.Field}
-              inputValue="true"
-            />
-            <Field
-              name="isTwoYearsNo"
-              label="5-årig master"
-              component={RadioButton.Field}
-              inputValue="false"
-            />
-          </RadioButtonGroup>
-          <RadioButtonGroup name="member" label="Vil du bli medlem av Abakus?">
-            <Flex>
-              <div
-                style={{
-                  marginLeft: '5px',
-                }}
-              >
-                <p className={styles.infoText}>
-                  Alle som går Kommunikasjonsteknologi & Digital Sikkerhet eller
-                  Datateknologi kan bli medlem av Abakus. Du må bli medlem for å
-                  kunne delta på arrangementer, kurs og annet Abakus tilbyr. Vi
-                  anbefaler alle nye studenter å melde seg inn. Du kan lese mer
-                  om{' '}
-                  <a
-                    href="https://abakus.no/pages/info-om-abakus"
-                    rel="noopener noreferrer"
-                    target="_blank"
-                  >
-                    Abakus
-                  </a>{' '}
-                  og{' '}
-                  <a
-                    href="https://statutter.abakus.no#medlemskap"
-                    rel="noopener noreferrer"
-                    target="_blank"
-                  >
-                    medlemskapet.{' '}
-                  </a>
-                  Det koster ingenting å være medlem av Abakus.
-                </p>
-              </div>
+        <h2>Verifiser studentstatus</h2>
+
+        {isStudent == null && (
+          <p>
+            For å kunne bli medlem i Abakus og få mulighet til å delta på
+            arrangementer, få tilgang til bilder og interessegrupper og mer må
+            du verifisere at du går enten Kommunikasjonsteknologi & Digital
+            Sikkerhet eller Datateknologi. Ved å trykke på knappen under gir du
+            Abakus tilgang til dine studier og fag i StudentWeb gjennom FEIDE
+            slik at vi kan registrere deg som medlem.
+          </p>
+        )}
+
+        {isStudent != null &&
```

Comment:
```suggestion
        {isStudent !== null &&
```

It seems you've intentionally removed the strict equality?

Reply:
I never get used to JS being JS :upside_down_face: